nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -345,3 +345,5 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 - QtMultimedia has changed its default backend to `QT_MEDIA_BACKEND=ffmpeg` (previously `gstreamer` on Linux or `darwin` on MacOS).
   The previous native backends remain available but are now minimally maintained. Refer to [upstream documentation](https://doc.qt.io/qt-6/qtmultimedia-index.html#ffmpeg-as-the-default-backend) for further details about each platform.
 
+- `services.postgresql.ensureUsers.*.ensureDBOwnership` now accepts a list of database names of which to set the owner besides setting it to `true` to set the owner of the database with the same name as the owner.
```

Comment:
Please sort this somewhere into the list like mentioned in the comment above the next headline.

Reply:
done